### PR TITLE
feat: adds action items to navbar (header)

### PR DESCRIPTION
**Pull request by jurajpiar**
Adds additional items to the header
Refactor- uses grid for layout

Resolves [RMKT-349](https://rsklabs.atlassian.net/browse/RMKT-349)

### CODE REVIEW

File: src/components/organisms/Header/HeaderDesktop.tsx [hunk 1]
```diff
@@ -43,20 +38,32 @@ const useStyles = makeStyles((theme: Theme) => ({
...
+            <NavLink to={hreflogo}>
+              <LogoNavbar />
+            </NavLink>
+          </Grid>
+          <Grid
+            item
+            xs={6}
+            container
+            justify="flex-start"
+          >
```
Comment (farinalvaro) [inline src/components/organisms/Header/HeaderDesktop.tsx:63]: I think `container` prop of `Grid` component should not be mixed with `item` prop. Items should be wrapped inside containers

File: src/components/organisms/Header/HeaderDesktop.tsx [hunk 2]
```diff
@@ -71,10 +78,32 @@ const HeaderDesktop: FC<HeaderProps> = ({ hreflogo, items, login }) => {
...
+              </ListItemIcon>
+            ))
+          }
+          </Grid>
+          <Grid
+            item
+            container
+            xs={1}
+            alignContent="center"
+          >
```
Comment (farinalvaro) [inline src/components/organisms/Header/HeaderDesktop.tsx:103]: Same comment about items and containers

File: src/components/organisms/Header/HeaderMobile.tsx [hunk 3]
```diff
@@ -98,21 +94,41 @@ const HeaderMobile: FC<HeaderProps> = ({ hreflogo, items, login }) => {
...
+                <NavLink to={hreflogo}>
+                  <LogoNavbar />
+                </NavLink>
+              </Grid>
+              <Grid
+                item
+                xs={4}
+                container
+                justify="flex-end"
+                alignContent="center"
```
Comment (farinalvaro) [inline src/components/organisms/Header/HeaderMobile.tsx:117]: Same comment about items and containers

File: src/components/organisms/Header/HeaderDesktop.tsx [hunk 4]
```diff
@@ -71,10 +78,32 @@ const HeaderDesktop: FC<HeaderProps> = ({ hreflogo, items, login }) => {
...
+            item
+            xs={4}
+            container
+            justify="flex-end"
+            alignContent="center"
+          >
+            {
+            !!itemsEnd?.length
+            && itemsEnd.map(({ icon, onClick }: ActionHeaderItemProps) => (
+              <ListItemIcon key={icon.key} {...{ onClick }}>
```
Comment (farinalvaro) [inline src/components/organisms/Header/HeaderDesktop.tsx:92]: Got this error compiling: 
```Type 'string | number | null' is not assignable to type 'string | number | undefined'.```

File: src/components/organisms/Header/HeaderDesktop.tsx [hunk 5]
```diff
@@ -71,10 +78,32 @@ const HeaderDesktop: FC<HeaderProps> = ({ hreflogo, items, login }) => {
...
-        <div className={classes.loginContainer}>
-          <Login />
-        </div>
+          </Grid>
+          <Grid
+            item
+            xs={4}
+            container
+            justify="flex-end"
+            alignContent="center"
```
Comment (farinalvaro) [inline src/components/organisms/Header/HeaderDesktop.tsx:87]: Same comment about items and containers

**Review by farinalvaro — CHANGES_REQUESTED** [review]
Just commented about a compiling error and a mix between `item` and `container` props of Grid component. Apart from that, looks great!

File: src/components/organisms/Header/HeaderDesktop.tsx (hunk 4 above)
Comment (jurajpiar) [reply src/components/organisms/Header/HeaderDesktop.tsx]: no longer an issue

**Review by farinalvaro — APPROVED** [review]
🎉

**Pull request merged**